docs/sftt/sftt-api-spec.md

```diff
@@ -1681,3 +1681,92 @@ If the site id specified is invalid.
 ##### `401 UNAUTHORIZED`
 
 If the calling user is not an admin.
+
+### Record Stewardship Activity
+
+`POST api/v1/protected/sites/:site_id/record_stewardship`
+
+Records a stewardship activity for the given site. Indicate `True` if the activity (watered, mulched, etc.) was completed, else `False`.
+
+#### Request Body
+
+```json
+{
+  "date": TIMESTAMP,
+  "duration": INT | NULL,
```

Comment:
I think we should indicate here the units we're using for duration. We can do that be changing the name of the key to be minutesDuration, if you're super opposed to that then we can just have a note here that specifies that duration should be counted as minutes

Reply:
Noted in duration is time in minutes